content/api/template-language.apib

```diff
@@ -666,9 +666,44 @@ Here is a curly: {{
 ```
 </REPL>
 
+### Escaping curly braces in amp-mustache templates
+
+AMP HTML email content may contain "amp-mustache" templates.  Most double and triple curly brace expressions
+within amp-mustache templates are automatically escaped by SparkPost such that the amp-mustache expressions
+are preserved upon delivery to email clients.
+
+There are certain expressions that SparkPost will *not* escape.  These include:
+* Expressions that contain the keyword "sparkpost"
+* Expressions that are already a call to one of the curly escape macros: opening_double_curly(), closing_double_curly(), opening_triple_curly(), and closing_triple_curly()
+* Expressions that contain use of "loop_var"
+* The "else" and "end" control statements
+
+In the following example, {{name}}, {{price}}, {{#cart_items}}, {{/cart_items}}, and {{^cart_items}} will be automatically
+escaped such that they are preserved as amp-mustache template expressions upon delivery of the email.
+On the other hand, {{sparkpost_name}} will be treated as a sparkpost templating expression since it contains the "sparkpost" keyword.
+
+```
+<amp-list src="https://ampbyexample.com/json/cart.json" layout="fixed-height" height="80">
+    <template type="amp-mustache">
+      <div id="cart">
+        Cart items for {{sparkpost_name}}:<br>
+        {{#cart_items}}
+        <div class="cart-item">
+            <span>{{name}}</span>
+            <span>{{price}}</span>
+          </div>
+        {{/cart_items}}
+        {{^cart_items}}
+          There are no featured products available. Please check back again later.
+        {{/cart_items}}
+      </div>
+    </template>
+  </amp-list>
+```
+
 ## Dynamic Content
 
-Dynamic content is content stored in a variable that contains expressions, statements, and links. Expressions inside of variables are **not** executed by default. Likewise, links are not tracked. To correctly render dynamic content, you must do the following:
+Dynamic content is content stored in a variable that contains expressions, statements, and links. Expressions inside of variables are **not** executed by default. Likewise, are not tracked. To correctly render dynamic content, you must do the following:
```

Comment:
> Likewise, are not tracked

Should be "Likewise, links are not tracked."

Reply:
blaming you and Chris, I just copied this stuff over lol.

Reply:
actually was probably me.  I've updated